Frontend/ui-customer/src/Screens/CustomerUtils/ReportIssueChatScreen/ScreenData.purs

```diff
@@ -0,0 +1,121 @@
+-}
+
+module Screens.ReportIssueChatScreen.ScreenData where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Screens.Types (ReportIssueChatScreenState)
+import Components.ChatView.Controller (config) as ChatView
+import Common.Styles.Colors (black700, black800, blue600, blue800, blue900, green200, grey700, grey800, grey900, transparentGrey, white900) as Color
+import PrestoDOM (Visibility(..))
+
+initData :: ReportIssueChatScreenState
+initData = {
+    data: {
+        tripId: Nothing,
+        selectedOption: Nothing,
+        categoryName: "",
+        issueId: Nothing,
+        categoryId: "",
+        messageToBeSent: "",
+        chatConfig: chatConfig',
+        recordAudioState: recordAudioState',
+        addImagesState: addImagesState',
+        addedImages: [],
+        viewImageState: { image : "", imageName : Nothing},
+        recordedAudioUrl: Nothing,
+        addAudioState: addAudioState',
+        uploadedImagesIds: [],
+        uploadedAudioId: Nothing,
+        options: [],
+        chats: [],
+        isResolved: false,
+        merchantExoPhone : Nothing,
+        showStillHaveIssue : false,
+        selectedRide : Nothing
+    },
+    props : {
+      showSubmitComp: false,
+      showImageModel: false,
+      showAudioModel: false,
+      showCallDriverModel: false,
+      showCallSupportModel: false,
+      showRecordModel: false,
+      showViewImageModel: false,
+      isPopupModelOpen: false,
+      isKeyboardOpen: false,
+      timerId : ""
+    }
+}
+
+type OptionType = {
+    optionName :: String,
+    optionId :: Number
+}
+
+chatConfig' = ChatView.config {
+                       userConfig {
+                         userName = "later",
+                         appType = "Customer"
+                       }
+                     , messages = []
+                     , sendMessageActive = false
+                     , vehicleNo = ""
+                     , suggestionsList = []
+                     , hint = "Message"
+                     , suggestionHeader = "Start Chat"
+                     , suggestionDelay = 750
+                     , emptyChatHeader = "Type something"
+                     , showHeader = false
+                     , showStroke = false
+                     , mapsText = "maps"
+                     , grey700 = Color.grey700
+                     , blue600 = Color.blue600
+                     , enableSuggestionClick = false
+                     , blue900 = Color.blue900
+                     , transparentGrey = Color.transparentGrey
+                     , green200 = Color.green200
+                     , grey900 = Color.grey900
+                     , grey800 = Color.grey800
+                     , blue800 = Color.blue800
+                     , white900 = Color.white900
+                     , showTextEdit = false
+                     , spanParent = true
+                     , black800 = Color.black800
+                     , black700 = Color.black700
```

Comment:
why we need this ? are we sending it to native ?

Reply:
this is need for chat view component. that is older component. but it is not necessary. it will need refactor. can we do that in another PR?